bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/patient/mapper/PatientResponseMapper.java

```diff
@@ -80,7 +84,18 @@ private void mapPersonAttributes(Patient patient, List<String> patientSearchResu
         String queriedPersonAttributes = patientSearchResultFields.stream()
                 .map(attributeName -> {
                     PersonAttribute attribute = patient.getAttribute(attributeName);
-                    return attribute == null ? null : formKeyPair(attributeName, attribute.getValue());
+                    if(attribute != null) {
+                        if("org.openmrs.Concept".equals(attribute.getAttributeType().getFormat())) {
+                            Concept concept = Context.getConceptService().getConcept(attribute.getValue());
+                            ConceptName fullySpecifiedName = concept.getFullySpecifiedName(Context.getLocale());
```

Comment:
I don't see much difference when using 'Context.getLocale()' and 'Context.getUserContext().getLocale()'. 'Context.getLocale()' internally calls 'getUserContext().getLocale()' method. 
https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/context/Context.java#L727